packages/web-components/src/components/cbp-action-bar/cbp-action-bar.scss

```diff
@@ -7,47 +7,44 @@
  --cbp-action-bar-color-background: var(--cbp-color-gray-cool-70);
  --cbp-action-bar-color-background-dark: var(--cbp-color-gray-cool-5);
 
- --cbp-action-bar-info-color: var(--cbp-color-text-lightest);
- --cbp-action-bar-info-color-dark: var(--cbp-color-text-darkest);
+ --cbp-action-bar-color: var(--cbp-color-text-lightest);
+ --cbp-action-bar-color-dark: var(--cbp-color-text-darkest);
   
 }
 
 [data-cbp-theme=light] cbp-action-bar[context*=dark]:not([context=light-always]),
 [data-cbp-theme=dark] cbp-action-bar:not([context=dark-inverts]):not([context=light-always]) {
   --cbp-action-bar-color-background: var(--cbp-action-bar-color-background-dark);
-  --cbp-action-bar-info-color: var(--cbp-action-bar-info-color-dark);
+  --cbp-action-bar-color: var(--cbp-action-bar-color-dark);
   
 }
 
 cbp-action-bar {
   display: flex;
   align-items: center;
   justify-content: right;
-  height: var(--cbp-space-14x);
+  min-height: var(--cbp-space-14x);
   width: 100%;
   padding: var(--cbp-space-3x);
+  color: var(--cbp-action-bar-color);
   background-color: var(--cbp-action-bar-color-background);
-  box-shadow: var(--cbp-shadow-level-3-up);
+
 
    & > *{
```

Comment:
you removed part of this CSS, but I don't think you need this still and I don't see a gap added.